### PR TITLE
Checked the colors in the figma project and created new color names.

### PR DESCRIPTION
**Pull request by Cray-code**
For discussion...
Once agreed, I can test all components for the approved colors.

### CODE REVIEW

File: src/colors.js [hunk 1]
```diff
@@ -16,3 +16,45 @@ export const colors = {
...
+  fileBackground: 'rgba(196, 196, 196, 0.5)', // file manager background
+  noteBackground: 'rgba(187, 134, 252, 0.08)', // create/edit note background
+  barBackground: 'rgba(255, 255, 255, 0.09), #121212', // top bar background, bottom bar background
+  frameHover: '#3C78C8', // :hover frame notes
+  // delete ??
+  States_white_overlay_Focus: 'rgba(255, 255, 255, 0.12)', // duplicate
+  Darkener_Background: 'rgba(0, 0, 0, 0.6)', // not used??
+  Primary_100: '#DBB2FF', // not used??
+  Elevation_Overlay_01dp: 'rgba(255, 255, 255, 0.05), #121212', // not used??
+};
```
Comment (DarthVanger) [inline src/colors.js:41]: Looks good to me

File: src/colors.js [hunk 2]
```diff
@@ -16,3 +16,45 @@ export const colors = {
...
+  // background
+  systemBarBack: '#121212', // system bar background
+  mainBackground: 'rgba(255, 255, 255, 0.07), #121212', // main background
+  menuBackground: 'rgba(255, 255, 255, 0.12), #121212', // menu item background
+  menuSelectBackground: 'rgba(255, 255, 255, 0.12)', // selected menu item background
+  fileBackground: 'rgba(196, 196, 196, 0.5)', // file manager background
+  noteBackground: 'rgba(187, 134, 252, 0.08)', // create/edit note background
+  barBackground: 'rgba(255, 255, 255, 0.09), #121212', // top bar background, bottom bar background
+  frameHover: '#3C78C8', // :hover frame notes
+  // delete ??
```
Comment (DarthVanger) [inline src/colors.js:36]: yep let's delete it :)

**Comment by Cray-code** [comment]
Found and replaced all colors in components and theme. The old colors are left to resolve merge conflicts. 
Branch rebased.

File: src/components/LogoutButton/LogoutButtonComponent.js [hunk 3]
```diff
@@ -1,15 +1,19 @@
-import React, { Component } from 'react';
-
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
+import { Typography } from '@material-ui/core';
+
+const useStyles = makeStyles({
```
Comment (DarthVanger) [inline src/components/LogoutButton/LogoutButtonComponent.js:6]: I guess this change wasn't supposed to be in this pull request?

File: src/components/NotesPage/NotesPageTopBar.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```
Comment (DarthVanger) [inline src/components/NotesPage/NotesPageTopBar.js:1]: Why this change?

File: src/components/NotesPage/NotesPageTopBar.js [hunk 5]
```diff
@@ -20,20 +20,21 @@ import * as Selectors from 'components/Uploads/UploadsSelectors';
...
 import { colors } from 'colors';
 
-const anchorPositionMenu = { top: 60, left: 0 };
-
 export const NotesPageTopBar = () => {
   const hasUploads = useSelector(Selectors.hasUploads);
   // const hasActiveUploads = useSelector(Selectors.hasActiveUploads);
 
   const [isMenuOpen, setIsMenuOpen] = React.useState(false);
+  const [anchorEl, setAnchorEl] = useState(null);
```
Comment (DarthVanger) [inline src/components/NotesPage/NotesPageTopBar.js:28]: I guess this change wasn't supposed to be in this pull request?

File: src/theme.js [hunk 6]
```diff
@@ -36,7 +29,16 @@ export const theme = createMuiTheme({
       fontSize: '14px',
       lineHeight: '24px',
       letterSpacing: 0.1,
-      color: 'rgba(255, 255, 255, 0.6)',
+      color: colors.textColor,
+    },
+    h6: {
+      fontFamily: 'Roboto',
```
Comment (DarthVanger) [inline src/theme.js:35]: How come you are adding `h6` to `theme.js` in this pull request, if we already have it in `master`?
https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L34

File: src/theme.js [hunk 7]
```diff
@@ -1,33 +1,26 @@
...
     },
     secondary: {
-      main: colors.turquoise,
-      button: colors.buttonGray,
+      main: colors.secondaryColor,
+      button: colors.titleColor,
     },
   },
   typography: {
-    headline6: {
```
Comment (DarthVanger) [inline src/theme.js:16]: I don't see `headline6` in `master` in `theme.js`: https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L34

How can this pull request delete from `master` something that is not present in `master`?
To be honest I don't understand how is that even possible :) Why is github showing this code as deleted.

Comment (DarthVanger) [reply src/theme.js:16]: In this pull request you have commit for removing `headline6`:

image

https://github.com/DarthVanger/NeuralNotes/pull/209/commits

But the same commit with the same number is in [master](https://github.com/DarthVanger/NeuralNotes/commits/master).

I don't understand why is github showing it as a new commit in this pull request.

On local when I do `git diff master` from this branch I don't see these changes shown by github.

Maybe try to create a new branch and a new pull request.

Or [this solution from stackoverflow](https://stackoverflow.com/a/46782679/1657101).

**Comment by Cray-code** [comment]
I was helped by temporarily changing the base to another branch and back.

**Review by DarthVanger — APPROVED** [review]
